OHS-Hosting-Infrastructure/complaint-tracker
# Sort complaints by grantee

## Description of change

Allows sorting loaded complaints by grantee name.

## Acceptance Criteria

Given that I am a program specialist
When I sort complaints by grantee
Then I see the complaints table, sorted by grantee name

## How to test

View complaints table, click on grantee column header, verify complaints sorted in alphabetical order by grantee. Click header again to reverse the sort

## Issue(s)

* closes #118 


## Checklist

To be completed by the submitter:



## To the Reviewer

This project is using [Conventional Comments](https://conventionalcomments.org/) to give structure
and context to PR comments. Please use these labels in your comments.

* **praise:**
* **nitpick:**
* **suggestion:**
* **issue:**
* **question:**
* **thought:**
* **chore:**
